Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04 as build
+
+LABEL org.opencontainers.image.authors="Santiago <san@san.com>"
+LABEL org.opencontainers.image.source="https://github.com/santaclose/ssbd"
+
+# hadolint ignore=DL3008
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+        binutils-mips-linux-gnu \
+        build-essential \
+        clang \
+        # FIXME: overkill
+        gcc-multilib \
+        git \
+        libcapstone-dev \
+        # NOTE: used by viewText.sh
+        nano \
+        # FIXME: just call python3
+        python-is-python3 \
+        python3 \
+        python3-pip \
```

Comment:
we can probably change the makefile to use python3

Reply:
I didn't want to interfere with your existing workflow but I can make the change.

Reply:
ok, i think it should work without `python-is-python3` already. can you test it? also would be cool not to have my name in this file pls. make those changes and i merge